lesson_16/objects/objects_app/src/main/java/com/codedifferently/lesson16/xaviercruz/Person.java

```diff
@@ -0,0 +1,95 @@
+package com.codedifferently.lesson16.xaviercruz;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Person {
+  private String name;
+  private int age;
+  private Position position;
+  private List<String> traits;
+  private Color eyeColor;
+  private final int max_traits = 10;
+
+  public Person(String name, int age, Position position, ArrayList<String> traits, Color eyeColor) {
+    this.name = name;
+    this.age = age;
+    this.position = position;
+    this.traits = traits;
+    this.eyeColor = eyeColor;
+  }
+
+  public Person(String name, int age, Position position, String[] traits, Color eyeColor) {
```

Comment:
Why do you have two constructors?

Reply:
The 2 constructors is to make a person with either a list of traits or an array of traits

Reply:
Now that i think of it creating a person with an ArrayList kind of makes no sense since i have a max_traits limit anyway.

Reply:
Removed the array constructor. i decided i like the builtin methods ArrayList provides.